src/Search/FilterConditions/SingleContextTitleFilterCondition.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+
+namespace App\Search\FilterConditions;
+
+
+use Elastica\Query\Terms;
+
+class SingleContextTitleFilterCondition implements FilterConditionInterface
+{
+    /**
+     * @var string $contextTitle
+     */
+    private $contextTitle;
+
+    /**
+     * @param string $contextTitle
+     * @return SingleContextTitleFilterCondition
+     */
+    public function setContextTitle(string $contextTitle): SingleContextTitleFilterCondition
```

Comment:
Vielleicht besser so ?

```suggestion
    public function setContextTitle(string $contextTitle): self
```